index.html

```diff
@@ -238,36 +274,30 @@ <h2 class="job-posts__title title">Popular Job Posts</h2>
 
               <!-- cards content middle -->
               <div class="card-content__middle">
-                <p class="card-content__middle_title white-txt">
-                  Sr. Product Designer
-                </p>
-                <p class="card-content__middle_subtitle white-txt">
+                <p class="card-content__middle_title">Sr. Product Designer</p>
```

Comment:
```suggestion
                <p class="card-content__title">Sr. Product Designer</p>
```

Why do you use a modifier naming? `title` is not a modifier for `middle`.

Reply:
Thanks for the comment, I didn’t notice that I wrote as a modifier. Renamed